README.md

```diff
@@ -1,6 +1,20 @@
-# README
+# pdc_discovery
 
-## Indexing a single handle
+A discovery portal for Princeton research data. Initially it will provide a better browsing experience for the research data containted in [DataSpace](https://dataspace.princeton.edu).
```

Comment:
`containted` was intended to be `contained` here

Reply:
Fixed! Thank you!